src/main/resources/com/google/api/codegen/java/grpc_test.snip

```diff
@@ -82,8 +82,10 @@
   public void {@test.name}() {
     {@addResponse(test)}
 
-    {@initCode(test.testCaseInitCode)}
+    @if test.testCaseInitCode.lines
```

Comment:
Why is this needed?

Reply:
To avoid the extra blank line if `test.testCaseInitCode` is empty.